agent/structs/structs.go

```diff
@@ -72,6 +72,43 @@ const (
 	SystemMetadataRequestType                   = 31
 )
```

Comment:
A comment right above this closing parenthesis would be useful as a reminder to update the map below